tbdex/rfq/rfq.go

```diff
@@ -21,20 +20,6 @@ type RFQ struct {
 	Signature       string                `json:"signature"`
 }
 
-// Sign cryptographically signs the RFQ using DID's private key
-func (r *RFQ) Sign(bearerDID did.BearerDID) error {
```

Comment:
bye sign. did we change our mind about whether there are use cases that require for signing a message after creation? Since `tbdex.Sign` still exists, I don't mind much either way, just curious if there's any broader context.

Reply:
I'm not sure, maybe @jiyoontbd or @mistermoe who create the issue have more context?
https://github.com/TBD54566975/tbdex-go/issues/29